concent_api/utils/helpers.py

```diff
@@ -2,14 +2,27 @@
 import base64
 import binascii
 import datetime
+import os
 import time
 
+from django.conf                    import settings
+from django.core.checks             import Error
 from django.utils                   import timezone
 
 from golem_messages                 import message
 from golem_messages.datastructures  import FrozenDict
 
 
+def storage_cluster_certificate_path_check(app_configs = None, **kwargs):
+    certificate_path = settings.STORAGE_CLUSTER_SSL_CERTIFICATE_PATH
+    if certificate_path != '':
+        if not os.path.exists(certificate_path):
+            return [Error("File not found")]
+        if os.path.splitext(certificate_path)[1] != '.crt':
+            return [Error(f"{certificate_path} is not a SSL certificate")]
```

Comment:
and what those should be?

Reply:
We don't have strict rules here. We just try to do what Django does in its own checks. For now the convention is `concent.XYYY` where `X` is `E` for errors and `W` for warnings. `Y` is a digit. Use subsequent numbers unless you want to reserve some range or numbers for a specific type of errors/warnings. It might be a good idea for each check to have a block of ten numbers for itself to avoid having to use a completely random number if you go back to an old check and want to add an error or a warning.

IDs can repeat if the same type of error/warning can appear in two different checks, though I'd generally err on the side of having too many codes rather than too few. For example right now we have `concent.E011` used twice and the two cases are kinda similar but not exactly so I'd rather have separate codes for them.

Reply:
done; Errors extracted to constants